src/tls.c

```diff
@@ -41,12 +42,12 @@ void tls_sni_inspection(
 
     if (strlen(sn) == 0) {
         log_print(PLATFORM_LOG_PRIORITY_INFO, "TLS server name not found");
-        return;
+        return 0;
     }
 
     log_print(PLATFORM_LOG_PRIORITY_INFO, "TLS server %s (%s) found", sn, dest);
 
-    sni_resolved(args, sn, dest);
+    return is_domain_blocked(args, sn, uid);
```

Comment:
This is just a nit, but we're mixing `jboolean` and `int` here. Would be good to clean up at some point (does not have to be now).

Reply:
Yeah, I am planning a general clean up plus testing...sigh!